.gitignore

```diff
@@ -158,3 +158,6 @@ logviewer/types.d.ts
 logviewer/pnpm-lock.yaml
 logviewer/webpack.config.js
 logviewer/webpack.generated.js
+
+### Mac Patch ###
+.DS_Store
```

Comment:
Why is this here? This has nothing to do with the PR.

Reply:
As discussed in the previous pr some Mac uses such as me keep being told to add those file so that's why it is there.

Reply:
1. You can disable them.
2. It should be in a different PR.

Reply:
I tried.

Reply:
If @Zabuzard he can delete it

Reply:
We had discussed this in the previous PR and decided to keep this.

Reply:
It seems to be common practice to put `.DS_STORE` on the gitignore. I agree, it should ideally be a separate PR. But it also doesnt harm much to have it here, so I did not bother too much.

The part you are definitely right about though, which is more important, is that it should be a separate commit. Otherwise it makes rebasing annoying. Ill extract it.

Reply:
Thanks illu, moved over to #245